package.json

```diff
@@ -11,7 +11,7 @@
     "@webassemblyjs/wasm-edit": "1.11.1",
     "@webassemblyjs/wasm-parser": "1.11.1",
     "acorn": "^8.4.1",
-    "acorn-import-assertions": "1.7.1",
+    "acorn-import-assertions": "1.7.2",
```

Comment:
Maybe in future better to use `^` to allow developers update locally without patch release?

Reply:
In the past we used to pin the dependencies, but happy to change

Reply:
let's look what @sokra answer, maybe it is not bad to have parser pinned, sometimes regressions happens

Reply:
^